packages/@o3r/components/schematics/ng-update/v10-0/index.spec.ts

```diff
@@ -74,4 +77,29 @@ describe('Update v10', () => {
       expect(tree.readText('src/components/example.template.html')).not.toMatch('| duration');
     });
   });
+
+  describe('Update c11n presenter$ declaration', () => {
+    it('shoud update the c11n presenter$ declaration', async () => {
+      const componentPath = 'src/components/example.component.ts';
+      initialTree.create(componentPath, `
+        import { Component } from '@angular/core';
+        import { O3rComponent } from '@o3r/core';
+        import { Observable } from 'rxjs';
+
+        @O3rComponent({ componentType: 'Component' })
+        @Component({
+          selector: 'o3r-example',
+          standalone: true,
+          templateUrl: './example.template.html'
+        })
+        export class ExampleComponent {
+          public presenter$!: Observable<MyComp>;
```

Comment:
could be nice to add another example that should not be impacted by the ng-update